src/core/src/bootstrap/Constants.py

```diff
@@ -282,6 +289,7 @@ class PatchAssessmentSummaryStartedBy(EnumBackport):
     class PatchOperationTopLevelErrorCode(EnumBackport):
         SUCCESS = 0
         ERROR = 1
+        WARNING = 2
```

Comment:
Was this verified end to end - that this will end up reflected in the API response? It _should_, but the question is just on verification. (note to self: come back to this)

Reply:
I have an ado item to modify the bvts to accept the warning status with start date of 2/12

done